Python/Product/Uwp/Templates/Projects/BackgroundService/PythonBackgroundService.pyproj

```diff
@@ -55,4 +55,9 @@
 
   <Import Project="$(PtvsTargetsFile)" />
 
+  <PropertyGroup>
+    <TargetPlatformVersion>$winsdkversion$</TargetPlatformVersion>
+    <TargetPlatformMinVersion>$winsdkminversion$</TargetPlatformMinVersion>
+  </PropertyGroup>
+
```

Comment:
Sounds like a recipe for breakage... I'd preserve the version at the top of your .targets and restore it at the end. That way you at least keep the opportunity to fix/change it in the future without having to modify user projects (we may have hit a problem like this before... :/ )